### ISSUE TITLE
PJ Link for Epson

### ISSUE THREAD

**Issue by AVKIWI** [bitfocus/companion-module-requests#1491]
hi got some of my PJ Link projectors working on companion bot others need a username and password to login to connect any help pleasse

**Comment by istnv** [comment]
PJLink protocol does not have a username, only a password. You may need to set this password via the Web control interface on the projector.

**Comment by AVKIWI** [comment]
Yeah is there something we can do to have username on it for projecters

The Epson projecters im trying to use are EB-X03 and EB-S140 I have the wifi cards in them I can do it off the Epsom projector software

I also have a EB925 which is working via the wifi unit currenty but its because this doesn’t require login

I have turn off the passwords but it is still asking for the username of EPSONWEB

Kind Regards
Logan Walker
Director
AVKIWI Limited|PO Box 40 598, Upper Hutt, Wellington|Phone 021-040-7760
***@***.******@***.***>

***@***.***

For all Serivce jobs please email ***@***.******@***.***>

Confidentiality: This message is intended for the use of the party to whom it is addressed and may contain information which is privileged and confidential. If you are not the intended recipient and have received this communication in error, you are notified that any use, disclosure, copying or distribution of the information is prohibited. If you have received this message in error, please notify us immediately, deleting the original message, please consider the envirionment before printing this email. Thank you.

**Comment by istnv** [comment]
Epson projector software does NOT use PJLink.
 PJLink does NOT have a username, only a password.
 I can send you the protocol document. 
 If you can find a username option, I will add it to the PJLink module.

**Comment by leonreucher** [comment]
I'll close this issue, since its module related and does not belong to the module request repo.

**Comment by byronhopp** [comment]
I am using the command ESC/VP.net to initialize where it requires a password within the same string all in HEX to allow for sending subsequent commands.  Does the "Epson Business Projector" module for Bit Focus Companion.  Howaver there is no question or textbox to enter it.  Is this module not aware of the new requirement, is their a version that works with the newer projectors?
